Doc/library/sqlite3.rst

```diff
@@ -521,6 +521,54 @@ Connection Objects
                  f.write('%s\n' % line)
 
 
+   .. method:: backup(target, *, pages=0, progress=None, name="main", sleep=250)
+
+      This method makes a backup of a SQLite database into the mandatory argument
+      *target*, even while it's being accessed by other clients, or concurrently by
+      the same connection. The *target* can be either a string, the path to the file
+      where the backup will be written, or another :class:`Connection` instance.
+
+      By default, or when *pages* is either ``0`` or a negative integer, the entire
+      database is copied in a single step; otherwise the method performs a loop
+      copying up to the specified *pages* at a time.
+
+      If *progress* is specified, it must either be ``None`` or a callable object that
+      will be executed at each iteration with three integer arguments, respectively
+      the *status* of the last iteration, the *remaining* number of pages still to be
+      copied and the *total* number of pages.
+
+      The *name* argument specifies the database name that will be copied: it must be
+      a string containing either ``"main"``, the default, to indicate the main
+      database, ``"temp"`` to indicate the temporary database or the name specified
+      after the ``AS`` keyword in an ``ATTACH`` statement for an attached database.
+
+      The *sleep* argument specifies the number of milliseconds to sleep by between
```

Comment:
Almost all CPython functions and methods use seconds as double for timeout. Milliseconds is going to confuse people.